### ISSUE TITLE
Unused Maven repository?

### ISSUE THREAD

**Issue by Rudloff** [andviane/google-books-android-viewer#2]
Hello,

We recently packaged your app in [F-Droid](https://f-droid.org/) but we don't allow untrusted Maven repositories so we had to remove the `oss.sonatype.org` repository: https://gitlab.com/fdroid/fdroiddata/blob/1a75a809e5b553e2bcab4238cf7d708165bb4b85/metadata/com.ames.books/uncover.patch

However, your app seems to build correctly without it. Is this repository really needed?

**Comment by andviane** [comment]
No, this is the repository where the library can be published after it builds. You do not need it to fetch any dependencies from there, this is the deployment target. I want library to be available for development via standard Gradle build scripts.
